internal/model/report_repository.go

```diff
@@ -26,7 +26,7 @@ func (r *ReportRepository) CreateTables() error {
 	if err == nil {
 		slog.Debug("Reports table created")
 	} else {
-		slog.Error("Create table error", slog.String("Error", err.Error()))
+		slog.Error("Create table error", "error", err)
```

Comment:
<img width="561" alt="image" src="https://github.com/Calindra/nonodo/assets/218768/ff61a89c-6fc9-4e84-8a5b-e18d0cad45a9">

Reply:
just to keep the simplicity of the original format
<img width="770" alt="image" src="https://github.com/Calindra/nonodo/assets/218768/3de5c662-8427-4e58-bc3d-e23222423e5f">

Reply:
golangci-lint have problems with this and CI break.

Reply:
For future reference:
![image](https://github.com/Calindra/nonodo/assets/6170125/d38f81fd-7388-47ae-b869-62a10834ac72)

Reply:
the lint is not complaining anymore.
You just need to put a label before the object.
slog.Error("message", "label", value)

Reply:
Maybe don't work, you can see what he see here, only `string` or `slog.Attr` and maybe `Interface`. Other types can don't work.

https://github.com/golang/tools/blob/master/go/analysis/passes/slog/slog.go#L115-L121